src/main.js

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
The formatting of this file is strange (no space between `if` and `(`), space between `Error (``. Hard ro read. Could you use some JS beautifier? Like https://beautifier.io/

Reply:
I have added `run lint` and `run lint-fix` commands to invoke linter. Surprisingly enough, it also fixes `let/const` mess in variable declarations.